sdk/samples/iot/hub/src/paho_iot_hub_c2d_example.c

```diff
@@ -149,35 +188,62 @@ static az_result read_configuration_and_init_client()
   return AZ_OK;
 }
 
-static int on_received(void* context, char* topicName, int topicLen, MQTTClient_message* message)
+// Read OS environment variables using stdlib function
+static az_result read_configuration_entry(
+    const char* name,
```

Comment:
do we really need "name" and "env_name"? Quick search shows these are always the same.

Reply:
nope!